packages/drivers/file-driver/src/fileDocumentStorageService.ts

```diff
@@ -76,8 +76,11 @@ export class FluidFetchReader extends ReadDocumentStorageServiceBase implements
      * @param versionId - version ID.
      * @param count - Number of versions to be returned.
      */
-    public async getVersions(versionId: string, count: number): Promise<api.IVersion[]> {
-        if (versionId === FileStorageDocumentName) {
+    public async getVersions(versionId: string | null, count: number): Promise<api.IVersion[]> {
+        if (versionId === null){
```

Comment:
I'd rather roll it into next IF. There is some logic there that may return [] that should apply as equally to null option as it does for FileStorageVersionTreeId